docs/api/MessengerPlugin.md

```diff
@@ -1,5 +1,5 @@
 <!-- Generated automatically, DO NOT EDIT! -->
-<a name="head.Messenger_Plugin"></a>
+<a name="Messenger_Plugin"></a>
```

Comment:
"head." and other things are auto-generated. Why are we editing an auto-generated file?

Reply:
The generated files are post-processed using a script that fixes the non-standard generated anchors. These anchors lead to broken links in our documentation site.